lines/producer/sender.py

```diff
@@ -0,0 +1,45 @@
+"""Kafka Publish Modules."""
+import json
+import os
+
+from kafka import KafkaProducer
+from prefect import Task
+
+from lines.libs.errors import MalformedPayloadError
+
+
+class EventProducer(Task):
+    """Event Producer Object."""
+
+    def __init__(self, *args, **kwargs):
+        self._server = os.getenv("KAFKA_BOOTSTRAP_SERVER")  # temporary
```

Comment:
Docstring:

```
"""
This handles producing new events to the Kafka pipeline.  Typically, these events will be xyz.

Attributes
----------
_server (str): Server address for Kafka.
_topic (str): Kafka topic for producing events.
"""
```

Reply:
Why use `_xyz`? These seem very important.

Reply:
the leading underscore is used as these are considered private instance variables

Reply:
Should they be considered private?